src/controllers/DataSetView.js

```diff
@@ -23,7 +23,13 @@ export default class DataSetView extends ViewController {
         morph: morph,
         rows: datarows,
         fields: (context.object.fields ? context.object.fields : []),
-        ...context
+        pagination: {pageLength: 50},

```

Comment:
Probably should be in config. Issue is that, since I'm using MVCController (which might have been a mistake) I don't have a way to pass options through to these controller classes. At least not yet.